index.html

```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><meta http-equiv="X-UA-Compatible" content="ie=edge"><title>freeCodeCamp Manila</title></head><body> <h1>Website is under construction!</h1> <h2>freeCodeCamp Manila</h2> <p> Want to contribute? Check our <a href="https://github.com/fccManila/fccmanila.github.io">Github</a> repo. </p> </body></html>
```

Comment:
This file is from the master branch which the build files, same with `script.75da7f30.js` and `script.75da7f30.js.map`. They are not included in develop branch you can delete this files.